pkg/model/initializers.go

```diff
@@ -243,7 +249,11 @@ func (ml *ModelLoader) GreedyLoader(opts ...Option) (grpc.Backend, error) {
 
 	// autoload also external backends
 	allBackendsToAutoLoad := []string{}
-	allBackendsToAutoLoad = append(allBackendsToAutoLoad, AutoLoadBackends...)
+	autoLoadBackends, err := bakends(o.assetDir)
```

Comment:
Just trying to understand if this is a change in functionality here or not: I think that we are still loading the same backends, just specifying them differently, correct?

Reply:
correct, the change here is just avoiding to specify the list manually, by scanning the asset directory which contains the backends extracted from the embedfs